.appveyor.yml

```diff
@@ -38,5 +38,5 @@ build: false
 test_script:
   - mkdir empty_folder
   - cd empty_folder
-  # eventually `- python -m pytest ../tests`
   - python -c "import httpstan"
+  - python -m pytest ../tests --deselect tests/test_bernoulli.py::test_bernoulli_parallel -v
```

Comment:
Let's run all the tests?

Reply:
Yes, I will change this after the "current test set"  works. So in 30min or so.